deploy/AzureAppServices/Datadog.Development.AzureAppServices.nuspec

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<package>
+  <metadata>
+    <id>Datadog.Development.AzureAppServices</id>
+    <version>1.10.40-prerelease</version>
+    <title>Unstable .NET Datadog APM</title>
+    <authors>Datadog</authors>
```

Comment:
+1 for listing `Datadog` as the author (unlike our other nuget packages). I like seeing my name in lights too, but better to show the company name.